src/main/kotlin/uk/gov/justice/digital/hmpps/assessrisksandneeds/api/controllers/RisksController.kt

```diff
@@ -0,0 +1,42 @@
+package uk.gov.justice.digital.hmpps.assessrisksandneeds.api.controllers
+
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.Parameter
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import io.swagger.v3.oas.annotations.responses.ApiResponses
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestMethod
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.assessrisksandneeds.api.model.RiskLevel
+import uk.gov.justice.digital.hmpps.assessrisksandneeds.api.model.RiskRoshSummaryDto
+
+@RestController
+class RisksController {
+
+  @RequestMapping(path = ["/risks/crn/{crn}/summary"], method = [RequestMethod.GET])
+  @Operation(description = "Gets rosh summary for crn")
+  @ApiResponses(
+    value = [
+      ApiResponse(responseCode = "403", description = "Unauthorized"),
+      ApiResponse(responseCode = "404", description = "CRN Not Found"),
+      ApiResponse(responseCode = "200", description = "OK")
+    ]
+  )
+  @PreAuthorize("hasAnyRole('ROLE_PROBATION', 'ROLE_CRS_PROVIDER') and hasAuthority('SCOPE_read')")
+  fun getRiskSummaryByCrn(
+    @Parameter(description = "CRN", required = true, example = "D1974X")
+    @PathVariable crn: String,
+  ): RiskRoshSummaryDto {
+    return RiskRoshSummaryDto(
+      "whoisAtRisk",
+      "natureOfRisk",
+      "riskImminence",
+      "riskIncreaseFactors",
+      "riskMitigationFactors",
+      mapOf(RiskLevel.HIGH to listOf("children")),
+      mapOf(RiskLevel.MEDIUM to listOf("known adult"))
+    )
+  }
```

Comment:
placeholder just to test the views filtering

Reply:
:thum